src/Dialog/Dialog.less

```diff
@@ -1,34 +1,27 @@
 /* Dialog.css */
 .moon-dialog {
-	padding: 24px 42px 42px;
-}
-.moon-dialog-title {
-	line-height: 1.3em;
-	margin-bottom: 6px;
-}
-.moon-dialog-client-wrapper {
-	min-height: 108px;
-}
-.moon-dialog-content {
-	margin: 0 0 0;
-}
-.moon-dialog-divider {
-	padding-bottom: 0;
-	border-bottom-width: 3px;
-	margin: 18px 0 18px;
-}
-.moon-dialog-client {
-	padding: 24px 0 0;
-	float: right;
-}
-.moon-dialog-client > * {
-	margin-left: @moon-grid-gutter-width;
-	margin-right: 0;
-}
-.enyo-locale-right-to-left .moon-dialog-client {
-	float: left;
-}
-.enyo-locale-right-to-left .moon-dialog-client > * {
-	margin-left: 0;
-	margin-right: @moon-grid-gutter-width;
+	padding-bottom: 0; // Override popup, so our optional components won't be stuck with too much padding
+
+	.moon-dialog-title-wrapper,
+	.moon-dialog-content,
+	.moon-dialog-client {
+		margin-bottom: 39px;
+	}
+
+	.moon-dialog-title-wrapper {
+		&.use-divider {
```

Comment:
Correct. We no longer want to use moon.Divider here. Dialog will now no longer require Divider. It's just an optional line now. Also, due to new GUI specs, they'd prefer this line to be 1px, not 3px.